der/src/asn1/any.rs

```diff
@@ -282,3 +280,17 @@ impl Tagged for Any {
         self.tag
     }
 }
+
+#[cfg(feature = "alloc")]
+impl<'a, T> From<T> for Any
+where
+    T: Into<AnyRef<'a>>,
+{
+    fn from(input: T) -> Any {
+        let anyref: AnyRef<'a> = input.into();
+        Self {
+            tag: anyref.tag(),
+            value: Bytes::new(anyref.value()).expect("invalid ANY"),
```

Comment:
Seems like you could write `impl From<ByteSlice<'_>> for Bytes` and use it here rather than `expect`?